project/src/store/api-actions.ts

```diff
@@ -0,0 +1,12 @@
+import { loadOffers } from './action';
+import { APIRoute } from '../const';
+import { OfferFromServer } from '../types/offer';
+import { ThunkActionResult } from '../types/action';
+import { Adapter } from '../utils/adapter';
+
+export const fetchOffersAction = (): ThunkActionResult =>
+  async (dispatch, _getState, api): Promise<void> => {
+    const { data } = await api.get(APIRoute.GetOffers);
```

Comment:
```suggestion
    const { data } = await api.get(APIRoute.GetOffers, {transformResponse : [(data) => data.map((offer: OfferFromServer) => Adapter.offerToClient(offer))]});
```
 должно работать не? https://github.com/axios/axios#request-config

Reply:
Не работаить
![image](https://user-images.githubusercontent.com/26679133/140817946-0ac8f951-22ac-4872-bc04-5d8311552cdd.png)

Reply:
Оки, я чекну надо глянуть что там приходит, и почему не работает.

Reply:
Ти набрехав, нічого ти не зробив!